java-squid/src/main/java/org/sonar/java/model/declaration/MethodTreeImpl.java

```diff
@@ -289,16 +290,23 @@ private boolean isPublic() {
 
   public boolean isAnnotatedOverride() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "isAnnotatedOverride" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)